iceoryx2-bb/container/src/slotmap.rs

```diff
@@ -576,16 +581,16 @@ pub mod details {
 
 /// A compile-time fixed-size, shared memory compatible [`FixedSizeSlotMap`].
 #[repr(C)]
-#[derive(Debug)]
-pub struct FixedSizeSlotMap<T, const CAPACITY: usize> {
+#[derive(Debug, ZeroCopySend)]
```

Comment:
Same here like in the `FixedSizeQueue` case. It is completely legal to use the `FixedSlotMap` in a process local context with non `ZeroCopySend` types.

Please implement `ZeroCopySend` explicitly without using the macro when `T: ZeroCopySend`.